packages/client/ui/react-ui/src/providers/auth/web3/WagmiAuthProvider.tsx

```diff
@@ -0,0 +1,25 @@
+import type React from "react";
+import { createConfig, http, WagmiProvider } from "wagmi";
+import { mainnet, sepolia } from "wagmi/chains";
+// todo: investigate implications of using react-query
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { metaMask, walletConnect } from "wagmi/connectors";
+
+const config = createConfig({
+    chains: [mainnet, sepolia],
+    connectors: [metaMask(), walletConnect({ projectId: "94ed8f7549329dad7be968888eec3688" })],
```

Comment:
wish it would make sense to hide this in an api call so we could change it easily but not sure it would be worth it

Reply:
yeah, that would be nice